README.md

```diff
@@ -396,25 +388,27 @@ git.rb
 index.html
 sample_homepage.html
 sample_series.html
-</pre>
-    
-<div id = "contribute"> <br> </div>
-<h1> <u> How to contribute </u> </h1>
 
-[Back to contents](#top)
-<br><br>
+```
+
+## How To Contribute
+
+[Back to contents](#list-of-contents)
+
+
 The work flow is the same as that of any other repository. 
-<br> (1) Fork / clone the repository.
-<br> (2) Create a new branch , say `my-changes` and make your changes in this branch.
-<br> (3) Commit your changes and send a Pull request (PR) to this repository.
-<br> Active contributors would be rewarded with the tag of "Collabrators"
-<br> Bug fixes , Issues , Issue solutions , Optimizations & Enhancements are always welcome.
+(1) Fork / clone the repository.
+(2) Create a new branch , say `my-changes` and make your changes in this branch.
+(3) Commit your changes and send a Pull request (PR) to this repository.
+Active contributors would be rewarded with the tag of "Collabrators"
+Bug fixes,Issues , Issue solutions , Optimizations , Enhancements are always welcome.
 
-<div id = "license"> <br> </div>
-<h1> <u> License </u> </h1>
 
-[Back to contents](#top)
-<br><br>
+
+## License
+
+[Back to contents](#list-of-contents)
+ 
 The MIT License (MIT)
 
 Copyright (c) 2016 Athitya Kumar
```

Comment:
Change this to `The MIT License Copyright (c) 2017 - [Athitya Kumar](https://github.com/athityakumar). Please have a look at the [LICENSE.md](LICENSE.md) for more details.`